eox_core/social_tpa_backends.py

```diff
@@ -0,0 +1,35 @@
+"""
+Extensions to the regular defined third party auth backends
+"""
+import logging
+
+from django.conf import settings
+from social_core.backends.open_id_connect import OpenIdConnectAuth
```

Comment:
This is a new dependency to eox-core, maybe it is a good idea to add that to the requirements

Reply:
Im thinking better to import it conditionally